nisystemlink/clients/dataframe/_data_frame_client.py

```diff
@@ -107,6 +107,17 @@ def get_table_metadata(self, id: str) -> models.TableMetadata:
         """
         ...
 
+    @json
+    @patch(_BASE_PATH + "/tables/{id}", args=(Path, Body))
+    def update_table_metadata(self, id: str, update: models.ModifyTableRequest) -> None:
```

Comment:
Regardless of what we go with for the method name, I think we should be consistent between `update_table_metadata` and `ModifyTableRequest` (update vs. modify). In the DFS OpenAPI document, we consistently use "modify".

Reply:
I'm going with `modify_table(s)`.